gas_v3_doc/readme.md

```diff
@@ -91,29 +91,33 @@ You can also verify that for new account ops, it will cost 2.5Gwei x 40664236 =
+<h2><b>Monitor RAM cost regularly and adjust storage_price to align with the current RAM/CPU cost.</b></h2>
 
-However, since action `updtgasparam` will re-define the gas costs for storage related operations, which might break the compatibility of existing dapps, this operation should rarely be called unless there is a significant divergence between the current RAM price and the RAM price used in the last `updtgasparam` operation. For example, if the RAM price has been increased from 0.3EOS/KB into 3EOS/KB, or decreased from 0.3EOS/KB to 0.05EOS/KB.
+For example, if RAM price has increased from 0.3EOS/KB to 0.6EOS/KB, we then change the `storage_price` which is 2x of the original using `setgasprices` action. This operation can be done frequently (for example once per 10 minutes) as RAM or CPU price change.
 
-- This is the command example if RAM price has increased to 0.6EOS/KB. 
+However, since action `updtgasparam` will re-define the gas costs for storage related operations, which might break the compatibility of existing dapps, this operation should rarely be called unless there is a significant divergence between the current RAM price and the RAM price used in the last `updtgasparam` operation. For example, if the RAM price has increased enough to cause the updated `storage_price` (on the last setgasprices action call) to be greater than the `overhead_price` (which typically would not be changed), then there is a slightly stronger motivation to call `updtgasparam` because there would be an explicit gas refund to account for the excess fees collected for the overhead costs. That explicit gas refund would not occur when `storage_price` < `overhead_price`. This isn't a big deal, but it can look strange to the user that the gas estimation requires a much higher gas limit on the transaction than the final gas usage (after gas refund) that is billed for the transaction.
+
+This is the command example if RAM price has increased to 0.6EOS/KB. 
 ```
 overhead_price=5000000000
 storage_price=5000000000
 ./cleos push action eosio.evm setgasprices "{\"prices\":{\"overhead_price\":${overhead_price},\"storage_price\":${storage_price}}}" -p eosio.evm
 ```
 
-- wait for 3 minutes until the new prices are effective
+wait for 3 minutes until the new prices are effective
+
+- verify the new gas consumption and gas fee (using gas price = 5Gwei, inclusion price = 0Gwei)
+  - Basic gas token transfer to new account: 40685236 gas (0.2034 EOS) (Notice that since `storage price` = `gas price` = 5Gwei, there is no discount to storage related gas consumption if `inclusion price` is set to zero, or in the case where transactions are initiated via the `call` action. It will still have some discounts for EVM transactions normally sent to the endpoint since the EVM miner enforces a positive minimum inclusion price.)
```

Comment:
Again, I wasn't telling you what to write verbatim. Just giving a comment to suggest a clarification might be helpful.

If you do want to rewrite this part, I would rephrase this as:

(Notice that since `inclusion_price = 0` and `storage price` = `gas price` = 5Gwei, there is no discount to storage related gas consumption.)